app/Http/Controllers/PrintController.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\User;
+use App\PrintAccount;
+
+class PrintController extends Controller
+{
+    public function index() {
+        return view('print.app');
+    }
+
+    public function modify_balance(Request $request) {
+        $printAccount = User::find($request->user_id)->printAccount;
+        $printAccount->balance += $request->balance;
+        $printAccount->save();
```

Comment:
Should use [increment](https://laravel.com/docs/5.8/queries#increment-and-decrement) for atomicity.

Reply:
Done.